### PR TITLE
Civ 1563 other expenses

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1563

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/assets/js/append-row.js [hunk 1]
```diff
@@ -4,13 +4,30 @@
...
     appendRowButton[0].addEventListener('click', (event) => {
       event.preventDefault();
       cloneRow();
+      showRemoveButton();
+      addEventListenerToRemoveButtons();
+    });
+    addEventListenerToRemoveButtons();
+  }
+
+  function removeRowButtonEventListener(element) {
```
Comment (LeonardoPalmeiro) [inline src/main/assets/js/append-row.js:22]: how could I test it?

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]
I've some things 
1) the add another expense button is just below in the month combobox
image
2) when we've error on other seems like the errors keep in all set of "Source of expense"
image
3) sometime the amount is no sum correctly like 100.23 and sometimes total stay with zero.

File: src/main/assets/js/append-row.js (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/assets/js/append-row.js:24]: the sreenshot was probably from not updated version and also fixed the calculation now and the layout. I hope the demo explained about the remove button :) if not i can show you again

**Comment by TatsyMaslova** [comment]
> I've some things
> 
> 1. the add another expense button is just below in the month combobox
>    image
> 2. when we've error on other seems like the errors keep in all set of "Source of expense"
>    image
> 3. sometime the amount is no sum correctly like 100.23 and sometimes total stay with zero.

As discussed these issues were from an older version. and the code was not quite up to date 😃

**Comment by TatsyMaslova** [comment]
oops accidentally closed, reopening

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by laarmada — APPROVED** [review]
Amazing job!!!

**Review by GorilaNaranja — APPROVED** [review]

**Pull request merged**